bluebottle/time_based/tests/test_admin.py

```diff
@@ -73,9 +73,24 @@ def test_list_activities_office(self):
         response = self.app.get(url)
         self.assertEqual(len(response.html.find_all("a", string="Schin op Geul")), 2)
 
+    def test_participant_count(self):
+        activity = DateActivityFactory.create()
+        DateParticipantFactory.create_batch(5, activity=activity)
+        url = reverse('admin:time_based_dateactivity_changelist')
+        response = self.app.get(url)
+        self.assertEqual(int(response.html.find_all('td')[5].get_text()), 5)
+
+    def test_start_date(self):
+        activity = DateActivityFactory.create()
+        DateActivitySlotFactory.create(activity=activity)
+        date = defaultfilters.date(activity.slots.all()[0].start) + ', '\
+               + defaultfilters.time(activity.slots.all()[0].start + timedelta(hours=2)) # noqa
```

Comment:
In this test you want to verify that the earliest slot is shown. But since you created an activity with a single slot. This test does not verify that at all. You will need to create at least 2 slots, and then make sure that the earliest is displayed. Either by creating a slot that you now is earlier then other slots. Or by sorting the slots and taking the first.